calculator_simple/README.md

```diff
@@ -0,0 +1,52 @@
+# Simple Calculator (Example) App
+
+This is an example of a GTK+3 application designed to function as a simple Calculator.
+
+## Prerequisites
+
+- GTK+ v3 installed on system.
+- gtkmm v3 installed on system.
+- C++ Compiler (modern).
+- GNU Make v3.82 or higher.
+
+## Tested System Setup
+
+- macOS Catalina (`10.15.4`)
+- Homebrew: [https://brew.sh/](https://brew.sh/)
+- GTK+3 install: `3.24.18` (macOS Homebrew)
+- gtkmm3 install: `3.24.2` (macOS Homebrew)
+- GCC install: `9.3.0_1` (macOS Homebrew)
+- GNU Make (`gmake`) install: `4.3` (macOS Homebrew)
```

Comment:
This line is fine, but the `brew` command to install GNU Make is missing from either this README or the top-level repo README.

The install is:
`brew install make`

And a note should be added that a warning will be printed explaining that the latest install of GNU Make is symbolically linked as `gmake` instead of `make`, to preserve/maintain XCode functionality in the macOS environment.

Reply:
Also need to add `cairomm` to docs now, since c14911d.